### DISCUSSION TITLE
How do I find my company_id?

### DISCUSSION THREAD

**Discussion by giuntim** [fattureincloud/api#11] [Q&A] [answered]
First of all, thank you for the new API, they are great and offer a wide range of possibilities to integrate external application.
Just I have difficulties in finding the company_id value to test my App. I would expect something like a "/info/companies" api but it does not exist.
How an I have a list of the companies I can operate with (given an auth token)?

Thx.

**Comment by giuntim** [comment, marked as answer]
ah! I found it, it's /user/companies . Well played! :-)

**Reply by valmoz** [reply]
hi @giuntim, we're glad that you like our new APIs. 

[This page](https://developers.fattureincloud.it/docs/basics/company-scoped-methods#-where-can-i-find-my-company_id) explains exactly how to find the Company ID: as you said, you should use the [List User Companies](https://developers.fattureincloud.it/api-reference/#get-/user/companies) method.

Let us know if you have further questions.

Thanks
Mauro
